documentation/src/main/asciidoc/stories/assembly_cache_configuration.adoc

```diff
@@ -12,6 +12,7 @@ For instance, you can configure how {brandname} encodes entries in your caches,
 
 include::{topics}/con_declarative_cache_configuration.adoc[leveloffset=+1]
 include::{topics}/ref_infinispan_caches.adoc[leveloffset=+2]
+include::{topics}/ref_infinispan_config.adoc[leveloffset=+2]
```

Comment:
Do we need to create an additional topic here or can we just update the examples in `ref_infinispan_caches.adoc` to include multiple cache definitions?